bugsnag-plugin-android-ndk/src/main/jni/bugsnag_ndk.c

```diff
@@ -529,6 +529,50 @@ JNIEXPORT void JNICALL Java_com_bugsnag_android_ndk_NativeBridge_updateMetadata(
   bsg_release_env_write_lock();
 }
 
+JNIEXPORT jobject JNICALL Java_com_bugsnag_android_ndk_NativeBridge_getSignalStackTrace(
```

Comment:
There are a couple of places in this method where new objects are created but not released, which can cause memory leaks. It'd be good to try and address these, for an example of how the project typically cleans up references this function is pretty good: https://github.com/bugsnag/bugsnag-android/blob/78adb64f793fa6d89f121047e779fcd4f0cbddd6/bugsnag-plugin-android-ndk/src/main/jni/bugsnag_ndk.c#L170

JNI reference: https://docs.oracle.com/javase/7/docs/technotes/guides/jni/spec/functions.html#global_local